tests/README.md

```diff
@@ -50,12 +51,6 @@ In the case of `@dfly_multi_test_args` each parameter configuration will create
 
 Parameters can use environmental variables with a formatted string where `"{<VAR>}"` will be replaced with the value of the `<VAR>` environment variable. Due to [current pytest limtations](https://github.com/pytest-dev/pytest/issues/349) fixtures cannot be passed to either of these decorators, this is currently the provided way to pass the temporary directory path in a CLI parameter.
 
-### Test Examples
-- **[blpop_test](./dragonfly/blpop_test.py)**: Simple test case interacting with Dragonfly
-- **[snapshot_test](./dragonfly/snapshot_test.py)**: Example test using `@dfly_args`, environment variables and pre-test setup
-- **[key_limit_test](./dragonfly/key_limit_test.py)**: Example test using `@dfly_multi_test_args`
-- **[connection_test](./dragonfly/connection_test.py)**: Example for testing running with asynchronous multiple connections. 
-
```

Comment:
I suggest dropping support for such a detailed README

If someone is making such a big contribution that needs a separate pytest, then he'll probably find a way to navigate in testing code...

Reply:
agreed :)

Reply:
I do think that we need to have detailed documentation on this, The readme file is the first place to look. And if you have a good examples that you can point to, why not? at least for me this i something that I always looking at first (the reeame file